### PR TITLE
reacting to telemetry rename

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Updating our repos for the change in aspnet/EventNotification#29.

Repos affected:
aspnet/Diagnostics@10fb0c11d29a220244eff62a72f4a282b9cd4cc5
aspnet/Hosting (this commit)
aspnet/Entropy@7e0b21fbb20b7320fce6798fcd81994738ff5de3
aspnet/Coherence@230f46045c9bc117a601d2cfaa60119f5cc589bc
aspnet/EntityFramework#3422
aspnet/MVC

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-    "projects": ["src","c:/github/testing/src","/home/kiran/github/testing/src"]
```
Comment (rynowak) [inline global.json]: @kichalla shame on you!

File: src/Microsoft.AspNet.Hosting/Internal/HostingEngine.cs [hunk 2]
```diff
@@ -88,9 +88,7 @@ public virtual IApplication Start()
             var logger = _applicationServices.GetRequiredService<ILogger<HostingEngine>>();
             var contextFactory = _applicationServices.GetRequiredService<IHttpContextFactory>();
             var contextAccessor = _applicationServices.GetRequiredService<IHttpContextAccessor>();
-#pragma warning disable 0618
```
Comment (rynowak) [inline src/Microsoft.AspNet.Hosting/Internal/HostingEngine.cs]: Do a quick search for `0618` and make sure you caught all of these (if you didn't already)

File: src/Microsoft.AspNet.Hosting/Internal/HostingEngine.cs [hunk 3]
```diff
@@ -98,12 +96,7 @@ public virtual IApplication Start()
                     httpContext.ApplicationServices = _applicationServices;
                     var requestIdentifier = GetRequestIdentifier(httpContext);
                     contextAccessor.HttpContext = httpContext;
-#pragma warning disable 0618
-                    if (telemetrySource.IsEnabled("Microsoft.AspNet.Hosting.BeginRequest"))
-                    {
-                        telemetrySource.WriteTelemetry("Microsoft.AspNet.Hosting.BeginRequest", new { httpContext = httpContext });
-                    }
-#pragma warning restore 0618
+                    diagnosticSource.Write("Microsoft.AspNet.Hosting.BeginRequest", new { httpContext = httpContext });
```
Comment (rynowak) [inline src/Microsoft.AspNet.Hosting/Internal/HostingEngine.cs]: Leave the IsEnabled checks.

File: src/Microsoft.AspNet.Hosting/Internal/HostingEngine.cs [hunk 4]
```diff
@@ -98,12 +96,7 @@ public virtual IApplication Start()
                     httpContext.ApplicationServices = _applicationServices;
                     var requestIdentifier = GetRequestIdentifier(httpContext);
                     contextAccessor.HttpContext = httpContext;
-#pragma warning disable 0618
-                    if (telemetrySource.IsEnabled("Microsoft.AspNet.Hosting.BeginRequest"))
```
Comment (Tratcher) [inline src/Microsoft.AspNet.Hosting/Internal/HostingEngine.cs]: Why did you remove the IsEnabled checks?

**Comment by rynowak** [comment]
:watch:

File: src/Microsoft.AspNet.Hosting/Internal/HostingEngine.cs (hunk 3 above)
Comment (JunTaoLuo) [inline src/Microsoft.AspNet.Hosting/Internal/HostingEngine.cs]: But those checks are done in the `Write` method anyways.

File: src/Microsoft.AspNet.Hosting/Internal/HostingEngine.cs (hunk 2 above)
Comment (JunTaoLuo) [inline src/Microsoft.AspNet.Hosting/Internal/HostingEngine.cs]: I have removed all of them from this repo.

**Comment by Tratcher** [comment]
Can you include direct links to the other commits?

**Comment by JunTaoLuo** [comment]
@rynowak Can you make the changes required for MVC?

**Comment by JunTaoLuo** [comment]
Can I get a ship it from someone for this? I'll squash commits when pushing to dev.

**Comment by rynowak** [comment]
:shipit:
